packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -14,7 +14,7 @@ import * as Classes from "../../common/classes";
 
 // amount in pixels that the content div width changes when truncated vs when
 // not truncated. Note: could be modified by styles
-const CONTENT_DIV_WIDTH_DELTA = 25;
+const CONTENT_DIV_WIDTH_DELTA = 26;
```

Comment:
This is also wrong, it comes from the right property on the div. There's a comment in sass, I'll put one here too.